server/api/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+
+
+# Create your models here.
+class User(models.Model):
+    userid = models.IntegerField()
+    discorduserid = models.CharField(max_length=20)
+    discordguildid = models.CharField(max_length=20)
+    ismod = models.BooleanField()
+
+    def __str__(self) -> str:
+        return str(self.userid)
+
+
+class Exile(models.Model):
+    exileid = models.IntegerField()
+    userid = models.ForeignKey(User, on_delete=models.CASCADE)
+    reason = models.TextField()
+    starttimestamp = models.DateTimeField()
+    endtimestamp = models.DateTimeField()
+    exilestats = models.IntegerField()
+
+    def __str__(self) -> str:
+        return f'User {self.userid} exiled for reason: "{self.reason}"'
+
+
+class Strike(models.Model):
+    strikeid = models.IntegerField()
+    userid = models.ForeignKey(User, on_delete=models.CASCADE)
+    reason = models.TextField()
+    createtimestamp = models.DateTimeField()
```

Comment:
think this should also include strike severity? @KingPieson to clarify

Reply:
I will take a look at the schema again. I there were some changes made since the commit I was looking at.

Reply:
updated to match current schema from main